format/repositories.bzl

```diff
@@ -96,6 +96,16 @@ def rules_format_dependencies():
         url = "https://github.com/google/google-java-format/releases/download/v1.17.0/google-java-format-1.17.0-all-deps.jar",
     )
 
+    RULES_JVM_EXTERNAL_TAG = "5.3"
+    RULES_JVM_EXTERNAL_SHA = "d31e369b854322ca5098ea12c69d7175ded971435e55c18dd9dd5f29cc5249ac"
+
+    http_archive(
```

Comment:
is it possible to just fetch jar files rather than use rules_jvm_external, like the java formatter above?

Reply:
there are like 30 dependencies here :( ofc I can try and manually list them all, but then updating will be a pain

Reply:
unfortunately scalafmt doesn't provide a fat jar like google-java-format

Reply:
What about writing a little script that curl's the sonatype registry or coursier package manager output, and produces the list of `http_jar` calls to copy-paste into here?

Reply:
The reason I'm asking is that WORKSPACE dependencies are a terrible mess and anything we add is likely to version skew with user code (Either they accidentally pick up our version that breaks their existing, or we accidentally use their version that doesn't work)

Reply:
hmm, maybe it would be simpler to have a script which calls coursier to download and assemble a single jar that we would check into the repository? What do you think?

Reply:
That makes the release artifact large, and most users don't have scala. I'd rather download the same files a maven user would.

Reply:
> What about writing a little script that curl's the sonatype registry or coursier package manager output, and produces the list of http_jar calls to copy-paste into here?

Done, @alexeagle!